java-codegen/opensearch-openapi.yaml

```diff
@@ -334,6 +342,9 @@ paths:
       operationId: indices.clear_cache.0
       x-operation-group: indices.clear_cache
       x-version-added: '1.0'
+      x-distributions-excluded:
```

Comment:
@Xtansia I understand the bias towards supporting AWS, but I honestly believe the spec has to be vendor neutral

Reply:
@reta I understand your concern, in my opinion the spec is vendor neutral, it's just not vendor agnostic. This information is not just useful for future documentation purposes but also for automated testing against any cluster distribution (opensource, AWS managed or otherwise). I for one would certainly welcome other vendors to annotate any APIs they do not support or aid in setting up automated testing. And of course open an issue over on [opensearch-api-specification](https://github.com/opensearch-project/opensearch-api-specification/issues) so we can discuss this more in depth if you like 😄